tests/benchmarks/current_consumption/nfc_idle/sysbuild.cmake

```
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(SB_CONFIG_SOC_NRF54H20)
+  if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+    message(FATAL_ERROR "REMOTE_BOARD must be set to a valid board name")
+  endif()
+
+  # Add remote project
+  ExternalZephyrProject_Add(
+      APPLICATION remote
+      SOURCE_DIR ${ZEPHYR_BASE}/../nrf/tests/benchmarks/power_consumption/common/remote_sleep_forever
```

Please use `SYSBUILD_NRF_MODULE_DIR` instead.
```suggestion
      SOURCE_DIR ${SYSBUILD_NRF_MODULE_DIR}/tests/benchmarks/power_consumption/common/remote_sleep_forever
```